test/playwright/utils/navigation.ts

```diff
@@ -184,16 +184,17 @@ export const goToSearchTerm = async (
     await searchInput.type(term)
     // Click search button
     // Wait for navigation
+    const searchLabel = dir === 'ltr' ? 'Search' : 'يبحث'
```

Comment:
It'd be great if we could pull these from the actual JSON files instead :thinking:

Reply:
Do you think finding elements by text is still the best way, even with i18n complications? It kind of made me wish for the simplicity of data-testid when editing these tests...

Reply:
I've updated the commented line, and the search type names object to use the actual JSON files.
Do you know if we can get the page's `dir` property from `Playwright` `page` object? Otherwise, we would have to pass `direction` to all the utility functions that locate page elements :(

Reply:
It's interesting. I think from an a11y perspective it's best to use the label strings because those most closely reflect how an actual user will identify the elements. If our e2e tests can't work with those, then it would indicate truly that a user may not be able to use it either.

On the other hand, we run into this issue where we have to copy the strings out of the translations... I'm really not sure :slightly_smiling_face: Both seem sub-optimal but one has a11y implications and the other is just annoying from a maintenance perspective, so I'm inclined to choose the one that has better a11y guarantees (which you've used here) even if it's more annoying to maintain.